### DISCUSSION TITLE
[A/L] (2021/22) Funzioni - Esercizio 27

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#267] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152317314-57eb6af3-f71b-4918-ac12-582c7c435c1a.png)

**Comment by LuginiAndrea** [comment]
$$f: \mathbf{-N} \rightarrow \mathbf{N}, f\left(x\right) = -x$$
$$g: \mathbf{N - 0} \rightarrow \mathbf{N - 0}, g\left(z\right) = z$$
$$\mathbf{-N} \cup \mathbf{N - 0}= \mathbf{Z}, \mathbf{-N} \cap \mathbf{N - 0} = \emptyset$$
Prendiamo ora $x_1 = -2 \in mathbf{-N}$ e $x_2 = 2 \in \mathbf{N}$
$$h\left(x_2\right) = g\left(x_2\right) = x_2 = 2$$
$$h\left(x_1\right) = f\left(x_1\right) = -x_1 = -\left(-2\right) = 2$$
$h$ non è iniettiva
